conf/test.yaml

```diff
@@ -0,0 +1,24 @@
+hosts:
+  apple:
+    name: 'apple-app'
+  pear:
+    -
+      name: 'pear-api'
+      payload:
+        image: "someimage"
+    -
+      name: 'pear-auth'
+      payload:
+        image: "someother"
+  peach:
+    name: 'peach-web'
+    payload:
+      extra: 'true'
+service:
```

Comment:
I still don't understand what this does

Reply:
Are you thinking that it should just be `endpoint` and `payload` as root config items? In my mind `service` encapsulated whatever service you were going to ping where `service.endpoint` was what endpoint and `service.payload` was the basic payload you would be passing along.